vm/interpreter/src/vm.rs

```diff
@@ -481,6 +481,11 @@ where
         Option<DefaultRuntime<'db, '_, DB, R, V, P>>,
         Option<ActorError>,
     ) {
+        let default_preignition =
+            setup_preignition_genesis_actors_testnet(self.store).unwrap_or_default();
+        let default_postignition =
+            setup_postignition_genesis_actors_testnet(self.store).unwrap_or_default();
```

Comment:
This logic should not be inside of the VM. Also, this is being called every time there is a send in the VM, terribly inefficient

Reply:
I think this should be passed into the VM::new() as a function pointer right? I believe that was the whole point of that change.

Reply:
Ahhh Okay so it would make more sense to keep it as part of the DefaultRuntime then

Reply:
Well, acutally I dont meant that directly. It should be called every time you do a get_vm_circulating_supply. and that is what should be passed as the funciton pointer

Reply:
The circ supply calculation should yes, this is not really related to that, and this function should _not_ be called every time that is called, only when it's not initialized